framework/db/mysql/QueryBuilder.php

```diff
@@ -23,8 +23,8 @@ class QueryBuilder extends \yii\db\QueryBuilder
      * @var array mapping from abstract column types (keys) to physical column types (values).
      */
     public $typeMap = [
-        Schema::TYPE_PK => 'int(11) NOT NULL AUTO_INCREMENT PRIMARY KEY',
-        Schema::TYPE_UPK => 'int(11) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY',
+        Schema::TYPE_PK => 'int(10) NOT NULL AUTO_INCREMENT PRIMARY KEY',
```

Comment:
In case of signed int doesn't minus take one space? So it's eleven in total for -2147483648.